tests/react-test-app/index.android.js

```diff
@@ -88,18 +98,28 @@ async function runTests() {
 
     // write the unit tests reports
     try {
-        await RNFS.writeFile('/sdcard/tests.xml', xmlString, 'utf8');
+        const resultsFile = RNFS.DocumentDirectoryPath + '/tests.xml';
+        await RNFS.writeFile(resultsFile, xmlString, 'utf8');
+        console.log(`Test results file path ${resultsFile}`);
+        console.log("Test Results ==============================================");
+        const lines = xmlString.split(/\r?\n/);
+        //lines.forEach(line => console.log(line));
+        lines.forEach(line => split(line).forEach(str => console.log(str)));
```

Comment:
IMO it is not obvious why we need to chop up the output here, I believe a comment would help.